src/SDKs/DevSpaces/Management.DevSpaces/Generated/SdkInfo_DevSpacesManagementClient.cs

```diff
@@ -19,10 +19,22 @@ internal static partial class SdkInfo
           {
               return new Tuple<string, string, string>[]
               {
-                new Tuple<string, string, string>("DevSpaces", "Controllers", "2018-06-01-preview"),
-                new Tuple<string, string, string>("DevSpaces", "Operations", "2018-06-01-preview"),
+                new Tuple<string, string, string>("DevSpaces", "ContainerHostMappings", "2019-01-01-preview"),
```

Comment:
@sanar-microsoft are you planning to release a stable version for REST API version `2018-06-01-preview`?
Also, please run `msbuild build.proj /t:build /p:Scope=SDKs\DevSpaces` and commit the changes to .props file

Reply:
@dsgouda We plan to deprecate this API soon. Right now these API versions are in preview.